tests/wpacceptance/PushMenuTest.php

```diff
@@ -25,7 +25,9 @@ public function testMenuItemHover() {
 
 		$I->click( '#wp-admin-bar-distributor a' );
 
-		$I->seeElement( '#distributor-push-wrapper .new-connections-list' );
+		$I->waitUntilElementVisible( '#distributor-push-wrapper .new-connections-list' );
+
+		$I->seeElement( '#distributor-push-wrapper .add-connection' );
```

Comment:
Can we add some additional tests or steps to verify that the sites menu loads correctly? I'm not sure this really verifies that.

Reply:
This should verify the menu is loading correctly. The previous test verifies the element `.new-connections-list` exists. In our case, that element is loaded via `ajax`, so we have to `waitUntilElementVisible`, which will timeout eventually if that element doesn't become visible (if the ajax request fails or no connections are returned). Assuming that element becomes visible, we then check if there's an `.add-connection` element, which is the container that holds each individual connection.

Reply:
👍 thanks for clarifying